plugins/modules/k8s_info.py

```diff
@@ -93,8 +93,10 @@
   community.kubernetes.k8s_info:
     kind: Pod
     label_selectors:
-      - app = web
+      - "app = {{ app_label_web }}"
```

Comment:
Would you mind adding this a as a separate example? Just want to make sure no users see the variable being set and assume it's an important piece of the `label_selectors` option

Reply:
Make sense to me. Thanks for the review.